schwab_converter.py

```diff
@@ -4,6 +4,14 @@
 from converter import Converter
 
 
+def _create_mapping(list1, list2):
```

Comment:
Please, use `expected_headers` and `actual_headers` instead of `list1` and `list2` to be more readable